reana_workflow_controller/rest/workflows.py

```diff
@@ -207,18 +204,13 @@ def get_workflows(paginate=None):  # noqa
                 workflow_response["session_type"] = workflow.interactive_session_type
                 workflow_response["session_uri"] = workflow.interactive_session
             if verbose:
-                reana_fs = fs.open_fs(SHARED_VOLUME_PATH)
-                if reana_fs.exists(workflow.workspace_path):
-                    absolute_workspace_path = reana_fs.getospath(
-                        workflow.workspace_path
-                    )
-                    disk_usage_info = get_workspace_disk_usage(
-                        absolute_workspace_path, block_size=block_size
-                    )
-                    if disk_usage_info:
-                        workflow_response["size"] = disk_usage_info[-1]["size"]
-                    else:
-                        workflow_response["size"] = "0K"
+                disk_usage_info = workflow.get_workspace_disk_usage(
```

Comment:
I know it was done like this before, but, couldn't we use the `summarize` option and get just one element? I think that if we do the simplification stated in https://github.com/reanahub/reana-db/pull/84/files#r492677834 this code should become simpler.

Reply:
Yes, indeed. Nice catch!